### PR TITLE
ENH Plot max and median long/short exposures

### PR DESCRIPTION
**Pull request by a-campbell**
Issue: https://github.com/quantopian/pyfolio/issues/218

screenshot 2015-12-03 18 21 13

### CODE REVIEW

**Comment by a-campbell** [comment]
A misleading part of this plot is the fact that median takes the average two position concentrations when there are an even number of longs or shorts.

File: pyfolio/plotting.py [hunk 1]
```diff
@@ -936,6 +936,38 @@ def show_and_plot_top_positions(returns, positions_alloc,
...
+    Plots the max and median of long and short position concentrations
+    over the time.
+
+    Parameters
+    ----------
+    positions : pd.DataFrame
+        The positions that the strategy takes over time.
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    **kwargs, optional
```
Comment (twiecki) [inline pyfolio/plotting.py]: Left-over.

**Comment by twiecki** [comment]
> A potentially misleading part of this plot is the fact that median takes the average two position concentrations when there are an even number of longs or shorts.

It should take the average between the two center values if there's an even number but that's fine and expected.

Otherwise LGTM.

File: pyfolio/pos.py [hunk 2]
```diff
@@ -97,6 +97,37 @@ def get_top_long_short_abs(positions, top=10):
     return df_top_long, df_top_short, df_top_abs
 
 
+def get_max_median_position_conc(positions):
```
Comment (twiecki) [inline pyfolio/pos.py]: `get_max_median_position_concentration`

**Comment by a-campbell** [comment]
Should be good to go now.

File: pyfolio/plotting.py (hunk 1 above)
Comment (twiecki) [inline pyfolio/plotting.py]: Still doesn't do anything with kwars below.

**Pull request merged**